app/views/simplified-account/settings/email-notifications/templates.njk

```diff
@@ -98,13 +98,6 @@
         </p>
       {% endif %}
     </div>
-
-    <div class="pay-email-custom-paragraph">
-      <h3 class="govuk-heading-m">What we mean by ‘description’</h3>
-      <p class="govuk-body">
-        You added a payment description when you created a new payment call to the Pay API. If you’re using a payment link, the description is taken from the payment link title.
-      </p>
-    </div>
```

Comment:
Should the equivalent section on lines 145-150 also be deleted?

Reply:
Good spot, fixed.